src/osdc/Objecter.cc

```diff
@@ -377,6 +385,11 @@ void Objecter::init()
     pcb.add_u64_counter(l_osdc_osdop_omap_del, "omap_del",
 			"OSD OMAP delete operations");
 
+    pcb.add_time_avg(l_osdc_lat_big_write, "lat_big_write", "latency of write ops >= 128KB");
```

Comment:
The 128KiB split is very arbitrary -- I would just rely on a single perf counter for reads and one for writes.

Reply:
What do you think of adding a configuration item here?

Reply:
I don't think anyone would ever touch it so it would again be arbitrary. Even the OSDs themselves don't segment op latencies into small and large buckets.

Reply:
How do you think it's more appropriate to implement it here？Can you give me some suggestions?

Reply:
I would just have 2 latency perf counters -- 1 for read ops and 1 for write ops.

Reply:
I have updated according to your suggestion. Please review,thanks.